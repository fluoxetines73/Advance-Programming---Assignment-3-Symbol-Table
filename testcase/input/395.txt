INSERT d number
INSERT y number
LOOKUP y
INSERT a string
PRINT
PRINT
INSERT c string random
RPRINT