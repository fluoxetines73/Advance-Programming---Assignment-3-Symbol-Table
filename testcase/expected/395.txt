Invalid: INSERT c string random